ansible/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Alexandre Ferreira < alex dot jorge dot m at gmail dot com >
+
+pkgname=ansible
+pkgver=2.9.7
+pkgrel=1
+pkgdesc='Radically simple IT automation platform'
+arch=('i686' 'x86_64')
+url='https://www.ansible.com'
+license=('GPL3')
+depends=('python3' 'python-yaml' 'python-jinja')
+provides=('python-ansible')
+optdepends=('sshpass: for ssh connections with password')
+makedepends=('tar' 'gcc' 'python3' 'python3-pip' 'libffi' 'libffi-devel' 'libcrypt' 'libcrypt-devel' 'openssl' 'openssl-devel')
```

Comment:
also rename all python3 parts. In makedepends you need to leave only devel parts of packages. Others you need to check and move to depends